Framework/Core/include/Framework/MessageContext.h

```diff
@@ -152,7 +156,7 @@ class MessageContext
   class AlignedMemoryResource : public pmr::FairMQMemoryResource
   {
    public:
-    AlignedMemoryResource(fair::mq::FairMQMemoryResource* other)
+    AlignedMemoryResource(fair::mq::MemoryResource* other)
```

Comment:
@dennisklein : you definitely miss some of the cases from my PR, e.g. this one

Reply:
it is covered in the `dennisklein:maint2` branch: https://github.com/FairRootGroup/FairMQ/pull/383.

Reply:
ah, ok, I see. Well then, with my PR I was able to compile it. The question is still. Can we make this gradually, or do we need to force-merge alidisti + O2 + DD?